app/views/includes/results_table.scala.html

```diff
@@ -27,18 +27,18 @@
             <th class="multi-results-header-date"><span class="hidden">hidden text for empty th</span></th>
             <th class="multi-results-header-text table-background">
                 @if(calculationResponse.calcType == CalculationType.SURVIVOR.toInt) {
-                    @Messages("gmp.total.entitlement")
+                <span aria-hidden="true">@Messages("gmp.total.entitlement")</span><span class="visuallyhidden">@Messages("gmp.result.first-table-total.value")</span>
```

Comment:
Can you format this to be 
```
<span aria-hidden="true">
  @Messages("gmp.total.entitlement")
</span>
<span class="visuallyhidden">
  @Messages("gmp.result.first-table-total.value")
</span>
```

it is more readable like this, do this for other visually hidden segments